packages/blade-mcp/src/server.ts

```diff
@@ -34,27 +42,27 @@ try {
     version: getPackageJSONVersion(),
   });
 
-  server.tool('hi_blade', hiBladeDescription, hiBladeSchema, hiBladeCallback);
+  server.tool(hiBladeToolName, hiBladeToolDescription, hiBladeToolSchema, hiBladeToolCallback);
 
   server.tool(
-    'create_new_blade_project',
```

Comment:
made this change since i had to use the tool call name in instrumentation as well